src/products/Product.java

```diff
@@ -95,4 +95,19 @@ public int getId() {
     public void setId(int id) {
         this.id = id;
     }
+
+    @Override
```

Comment:
Non c'è bisogno di override

Reply:
perchè vuoi usare il metodo print ? io sinceramente trovo comodo poter fare i test direttamente sulle liste che mi interessano quindi se non vi dispiace lo vorrei l'override to string in product